openstack/identity/v3/projects/results.go

```diff
@@ -200,3 +200,21 @@ func (r ModifyTagsResult) Extract() (*ProjectTags, error) {
 type DeleteTagsResult struct {
 	gophercloud.ErrResult
 }
+
+// GetProjectTag is the result of a Get tag request. Call its Extract method to
+// interpret it.
+type CheckTagResult struct {
```

Comment:
Wouldn't it be cool to have `CheckTagResult` implement an `Extract` method that returns a bool and an error? A 404 should result in `(false, nil)`.